gatsby-node.js

```diff
@@ -91,58 +112,98 @@ exports.createPages = ({ actions, graphql }) => {
         }
       }
     }
-  `).then(result => {
+  `).then((result) => {
     if (result.errors) {
       return Promise.reject(result.errors);
     }
     const { edges } = result.data.allMarkdownRemark;
-    const allowedCategories = ['update', 'blog post']
+    const allowedCategories = [
+      "update",
+      "blog post",
+      "technology",
+      "engineering",
+      "security",
+      "community",
+    ];
 
     edges.forEach(({ node }) => {
-      node.frontmatter.categories.forEach(category => {
-        if(!allowedCategories.includes(category.toLowerCase())) {
-          return Promise.reject(`Category - ${category} is not allowed in the posts`);
+      node.frontmatter.categories.forEach((category) => {
+        if (!allowedCategories.includes(category.toLowerCase())) {
+          return Promise.reject(
+            `Category - ${category} is not allowed in the posts`
+          );
         }
-      })
-    })
+      });
+    });
 
     createPaginatedPages({
       createPage,
       edges,
       pageTemplate: path.resolve(`src/templates/blog.js`),
       paginatePost: "/blog", // old field. not remove
       pathPrefix: "/blog", // new field. not remove
-      pageLength: 8
+      pageLength: 8,
     });
 
     edges.forEach(({ node }) => {
       createPage({
         path: node.frontmatter.path,
         component: postTemplate,
-        context: {}
+        context: {},
       });
     });
 
     const tags = _(edges)
-      .map((item) => _.get(item, ['node', 'frontmatter', 'tags'], []))
+      .map((item) => _.get(item, ["node", "frontmatter", "tags"], []))
       .flatten()
-      .map((item) => item.toLowerCase().replace(' ', '-'))
+      .map((item) => item.toLowerCase().replace(" ", "-"))
       .uniq()
       .value();
 
-    tags.forEach(tag => {
-      const filteredEdges = edges.filter(({ node }) => node.frontmatter.tags.map(tag => tag.toLowerCase().replace(' ', '-')).includes(tag))
+    tags.forEach((tag) => {
+      const filteredEdges = edges.filter(({ node }) =>
+        node.frontmatter.tags
+          .map((tag) => tag.toLowerCase().replace(" ", "-"))
```

Comment:
`.toLowerCase().replace(" ", "-")` is repeated several times in this PR, probably I'd move it out to utils for easy reusing.